code/game/objects/items/AI_modules.dm

```diff
@@ -365,7 +365,7 @@ AI MODULES
 	var/subject = "human being"
 
 /obj/item/aiModule/core/full/asimov/attack_self(var/mob/user as mob)
-	var/targName = stripped_input(user, "Please enter a new subject that asimov is concerned with.", "Asimov to whom?", subject)
+	var/targName = stripped_input(user, "Please enter a new subject that asimov is concerned with.", "Asimov to whom?", subject, MAX_NAME_LEN)

```

Comment:
```suggestion
	var/targName = stripped_input(user, "Please enter a new subject that asimov is concerned with.", "Asimov to whom?", subject, CONFIG_GET(number/max_law_len))
```

Reply:
Should that one be set to the law length over name length? Other similar boards like the One Human use max name.

Reply:
reading... in 2019?